packages/react/src/components/Tooltip/Tooltip.module.css

```
@@ -0,0 +1,7 @@
+.wrapper {
+  background: var(--fds-semantic-border-neutral-strong);
+  padding: var(--fds-spacing-1) var(--fds-spacing-2);
+  color: white;
+  border-radius: var(--fds-spacing-1);
+  font-size: var(--fds-typography-paragraph-xsmall);
```

The typography tokens are made for use with [font](https://developer.mozilla.org/en-US/docs/Web/CSS/font) shorthand.
```suggestion
  font: var(--fds-typography-paragraph-xsmall);
```